arroyo-state/src/parquet.rs

```diff
@@ -30,11 +30,11 @@ use tokio::fs::{remove_file, File};
 use tokio::io::{AsyncReadExt, AsyncWriteExt};
 use tokio::sync::mpsc::{self, Receiver, Sender};
 use tokio::sync::oneshot;
-use tracing::log::warn;
+use tracing::warn;
 use tracing::{debug, info};
 
-pub const S3_STORAGE_ENGINE_ENV: &str = "S3_BUCKET";
-pub const S3_REGION_ENV: &str = "S3_REGION";
+pub const S3_STORAGE_ENGINE_ENV: &str = "S3_STORAGE_BUCKET";
```

Comment:
Why is this a different env var than the user setting?

Reply:
Could imagine them being different, but probably not worth it right now. I'll put it back to S3_REGION